router.go

```diff
@@ -30,10 +31,13 @@ func (r *Route) Call(req *http.Request) interface{} {
 	}
 
 	handler := r.matching(rs)
+	if handler == nil {
+		return nil, errors.New("page not found")
```

Comment:
`errors.New("page not found")` this is not good enough for me, I will say create error constants is better.

Reply:
you're right.